pkg/template/template.go

```diff
@@ -72,7 +73,9 @@ func (p *Processor) Process(template *api.Template) field.ErrorList {
 func stripNamespace(obj runtime.Object) {
 	// Remove namespace from the item
 	if itemMeta, err := meta.Accessor(obj); err == nil {
```

Comment:
Actually what's the problem here? We want to strip the namespace so `itemMeta.SetNamespace("")` should be fine to run always, no?